tests/blackbox/stratis_cli_cert.py

```diff
@@ -113,6 +115,26 @@ def setUp(self):
         time.sleep(1)
         exec_command(["udevadm", "settle"])
 
+    def test_permissions(self, command_line, permissions, exp_stdout_empty):
+        """
+        Test running CLI commands with and without root permissions.
+        :param list command_line: The arguments needed to execute the Stratis command being tested
+        :type command_line: List of str
+        :param bool permissions: True if the Stratis command needs root permissions to succeed,
+                                    otherwise False.
+        :param bool exp_stdout_empty: True if stdout is expected to be empty
+                                        when Stratis command succeeds.
+        """
+        os.seteuid(_NON_ROOT)
+
+        if permissions:
+            self.unittest_command(command_line, 1, False, True)
+        else:
+            self.unittest_command(command_line, 0, True, exp_stdout_empty)
+
+        os.seteuid(_ROOT)
```

Comment:
I think that, if the command should have failed because it didn't have root permissions, after the effective UID is reset to root is a good time to try to rerun the command. It should succeed in that case, or else the test is still a failure.

Reply:
Could I get additional clarification on this? If I understand what you're saying correctly, I believe I rerun the command as root on the line below.

Reply:
You got that right. I just somehow didn't see what I was looking at.